Ambit/Source/Ambit/Actors/SpawnerConfigs/SpawnOnPathConfig.h

```diff
@@ -28,13 +28,11 @@ struct AMBIT_API FSpawnOnPathConfig : FSpawnerBaseConfig
 
     bool bSnapToSurfaceBelow = false;
 
-    TEnumAsByte<EMatchBy> MatchBy = EMatchBy::NameAndTags;
+    TEnumAsByte<EMatchBy> MatchBy = NameAndTags;
```

Comment:
We should keep it. (From a long time ago conversation when its implementation)

Reply:
Will revert.